x-pack/plugins/security_solution_serverless/server/types.ts

```diff
@@ -65,17 +66,13 @@ export interface UsageMetrics {
   quantity: number;
   period_seconds?: number;
   cause?: string;
-  metadata?: unknown;
+  metadata?: ResourceSubtypeCounter;
```

Comment:
It feels a bit strange to import from a child file upward. We should move `ResourceSubtypeCounter` into this file or just use `[key: string]: string` directly.

Reply:
Done:)